airflow/providers/odbc/hooks/odbc.py

```diff
@@ -211,3 +215,9 @@ def get_sqlalchemy_connection(
         engine = self.get_sqlalchemy_engine(engine_kwargs=engine_kwargs)
         cnx = engine.connect(**(connect_kwargs or {}))
         return cnx
+
+    def _make_serializable(self, result: list[pyodbc.Row]) -> list[tuple] | list[pyodbc.Row]:
```

Comment:
nitpick: Should the return type be annotated this one?

```suggestion
    def _make_serializable(self, result: list[pyodbc.Row]) -> list[tuple[pyodbc.Row]] | list[pyodbc.Row]:
```

Reply:
No, when the result is made serializable, Row objects are transformed into pure Python data structure (tuples). Thus, in this case, it should either return a list of tuple, or return directly the given data.